src/vm/arm/asmhelpers.S

```diff
@@ -561,6 +561,15 @@ ThePreStubPatchLabel:
 
         .endm
 
+#ifdef PROFILING_SUPPORTED
+
+//
+// EXTERN_C void JIT_ProfilerEnterLeaveTailcallStub(UINT_PTR ProfilerHandle)
+//
+LEAF_ENTRY JIT_ProfilerEnterLeaveTailcallStub, _TEXT
+    bx lr
```

Comment:
Is this a temporary/placeholder thing? Or is this ever used in a full implementation?

Reply:
It's used if the profiler wants to only have some callbacks and not others (i.e. if they just want enter callbacks then we will register the taillcall stub and leave stub that just do rets for the other two).